planning/development-strategy.md

```diff
@@ -1,39 +1,142 @@
-# Restaurants finder
+# Job - Advisor
+
+<!---minju edited--->
+
```

Comment:
@minjupgeorge please change the description of the project so it's not only for IT sector.

Reply:
done